platform-tests/src/platform/acceptance/init_test.go

```diff
@@ -41,7 +41,9 @@ func TestSuite(t *testing.T) {
 	RegisterFailHandler(Fail)
 
 	testConfig, err = config.NewCatsConfig(os.Getenv("CONFIG"))
-	Expect(err).NotTo(HaveOccurred())
+	if err != nil {
```

Comment:
what is the reason for this change?

Reply:
as it was explained the Ginkgo/Gomega suite is not initialised at this point yet and the Expect will fail miserably.